raiden/tasks.py

```diff
@@ -122,7 +124,7 @@ def _run(self):  # pylint: disable=method-hidden
 
                     try:
                         self.callback(originating_contract, event)
-                    except:
+                    except:  # pylint: disable=bare-except
```

Comment:
It wasn't part of your commit, but I believe we should also log type/message of the exception